com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -443,6 +444,27 @@ public ScriptableRenderer GetRenderer(int index)
             return m_Renderers[index];
         }
 
+        /// <summary>
+        /// Returns a renderer data from the current pipeline asset
+        /// </summary>
+        /// <param name="index">Index to the renderer data. If invalid index is passed, the default renderer is returned instead.</param>
+        /// <returns></returns>
+        public ScriptableRendererData GetRendererData(int index)
+        {
+            if (index == -1)
```

Comment:
Why the case that I do `GetRenderData(-2)` returns `null` and if I do `GetRenderData(-1)` gives me the default?

In which cases this would happen?

Also if -1 is a special case to return the default one, we can add it as default value of the argument, this will allow us to get the default by simply calling the method.